lib/Importer.js

```diff
@@ -154,24 +154,32 @@ class Importer {
     const unusedImportVariables = new Set();
     Object.keys(unusedVariables).forEach((variable) => {
       const lineNumbers = unusedVariables[variable];
-      if (anyNumbersWithinRange(lineNumbers, oldImports.range)) {
+      if (oldImports.range.length > 1 &&
```

Comment:
If the buffer looks like this:

```
var span = <span/>;
```

`oldImports.range` will be `[0]`, even though there are no imports. Which led us to think that `span` was an import, even though it wasn't. There's probably a better fix for this, because I realize that this won't work if the file consists of only one import, and nothing else.

```
const foo = require('foo');
```